contrib/fwupd.spec.in

```diff
@@ -208,6 +208,8 @@ mkdir -p --mode=0700 $RPM_BUILD_ROOT%{_localstatedir}/lib/fwupd/gnupg
 %{_unitdir}/fwupd.service
 %{_unitdir}/system-update.target.wants/
 %dir %{_localstatedir}/lib/fwupd
+%dir %{_datadir}/fwupd/quirks.d
```

Comment:
Just a thought, can rpm packaging let you just wildcards to pick everything like `%dir %{_datadir}/fwupd/*`?  It should let this have to change less when you shuffle stuff around if you can.

Reply:
Yes, although IIUC it's best practice to list them individually, which means you notice if they go away or get renamed. You could argue the file-list could just be `*` :)

Reply:
But considering you're the upstream and you know when files come and go as a result, is that still a relevant best practice?  Just seems like paperwork at that point....

Reply:
I think using logic with the Fedora Packaging Policy is a dangerous game :) The other thing %dir lets you do is choose the permissions or group ownership, although in this case the quirks are not security sensitive and can use the default.

Reply:
Ah well I guess this is a difference from debian approach when it comes to policy.  `dh_fixperms `will run at set policy on everything unless you override it.

```
       dh_fixperms is a debhelper program that is responsible for setting the permissions of files and directories in package build directories to a sane state -- a state that complies with Debian
       policy.
```

I guess you know fedora policy and such better than me on what would fly with this kind of stuff.  I just think TPS reports are overrated and try to avoid them when I can.